### DISCUSSION TITLE
Why 3 different apps in the "apps" folder.

### DISCUSSION THREAD

**Discussion by Alvazz** [PhilDL/remix-gospel-stack#197] [Q&A]
Why are there 3 different projects in the apps directory, nextjs-app, remix-app, and remix-vercel? What are the differences between them? If I use this scaffolding to develop a project, will all 3 projects be started at the same time?

**Comment by PhilDL** [comment]
Hi, these apps are given as example starting point, feel free to delete any of them.

In my opinion there is 2 ways to use a monorepo setup:
- For a big project, bigger team of people, there would be different packages, sometimes owned by different teams. The apps folder would contain the main app, and sometimes different things like a Marketing website (could be a Remix app or a NextJS, or Astro app), a Docs website, etc...
- For a very small agency or a single freelancer, packages would contain reusable pieces of logic, like file upload, CSV parsing, email sending. These reusable packages would then get plugged into different apps inside the app folder. Each apps are different customer projects, in production. 

Probably other examples but that's the two that quickly come to mind. 

You could run them all at the same time, yes but you would have to double-check that they are not using the same port.
